Homework 3/script.js

```diff
@@ -39,12 +51,56 @@ $( document ).ready(function() {
 	});
 
 	$("#signUpButton").click(function() {
+		$("#signUpErrorMessage").hide();
+		$("#signUpErrorMessage").empty();
+		if(($('#signUpPasswordInput').val() != '') && ($('#signUpPasswordConfirmInput').val() != '') && ($('#signUpEmailInput').val() != '') && ($('#signUpUsernameInput').val() != '')){
```

Comment:
if statements should really never be this deep. That typically means you can probably refactor it. If you were to step back and read this code in a few months, you'd probably be confused.